django_projects/locallibrary/catalog/views.py

```diff
@@ -1,3 +1,17 @@
 from django.shortcuts import render
+from catalog.models import Book, Author, BookInstance, Genre
 
-# Create your views here.
+def index(request):
+    num_books = Book.objects.all().count()
```

Comment:
bỏ bớt code dư thừa nhé
```suggestion
    num_books = Book.objects.count()
```